operator/internal/common/labels/labels.go

```diff
@@ -13,6 +13,7 @@ import (
 
 var defaultLabels = map[string]string{
 	"app.stackrox.io/managed-by": "operator",
+	"app.stackrox.io/watched-by": "operator", // for resources managed by other components but watched by the Operator
```

Comment:
@porridge wdyt?

Reply:
Ah, so you want to flip to a new label? I thought we'd only add it on the CA bundle configmap created by sensor. Or are you saying it's not possible to watch an union of labels, just an intersection?